libpldmresponder/fru.cpp

```diff
@@ -441,6 +862,581 @@ int FruImpl::setFRUTable(const std::vector<uint8_t>& fruData)
     return PLDM_ERROR_UNSUPPORTED_PLDM_CMD;
 }
 
+void FruImpl::subscribeFruPresence(
+    const std::string& inventoryObjPath, const std::string& fruInterface,
+    const std::string& itemInterface,
+    std::vector<std::unique_ptr<sdbusplus::bus::match::match>>& fruHotPlugMatch)
```

Comment:
you can obtain this from PDI "sdbusplus::bus::match::match".

Reply:
match_t

Reply:
did this PDI change and PDRs got messed up , will change this after a discussion

Reply:
changed to match_t